app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.tsx

```diff
@@ -201,9 +205,15 @@ export const FormAudience = ({
             <Select
               placeholder="All Countries"
               isMulti
+              isDisabled={!isDesktop}
               {...formSelectAttrs("countries", setCountries)}
               options={selectOptions(config.countries as SelectIdItems)}
             />
+            {!isDesktop ? (
+              <p className="text-secondary">
```

Comment:
nit: is there any sort of `text-error` that we could use here?

Reply:
Oh yes there is, but I like to reserve using 'error' styled messages for cases where the user has entered something invalid they need to change, whereas here there's nothing they can do wrong we're just notifying them of why it's not available.  Generally we've used 'text-secondary' for notices like that in both old and new Experimenter, but I'm not opposed to rethinking how we use typography to signal different kinds of messages to users but I would hazard we approach that as a separate body of work.